testProgs/procedure.mp.asm

```diff
@@ -0,0 +1,16 @@
+BR L0
+BR L1
```

Comment:
Is this generated? Because if so, it is wrong. We do not need to BR to the procedure. This line will never be hit.